etc/httpd/conf.d/status.conf

```diff
@@ -0,0 +1,10 @@
+ExtendedStatus on
+<VirtualHost 127.0.0.1:80>
+    ServerName local-stackdriver-agent.stackdriver.com
+    <Location /server-status>
+        SetHandler server-status
+        Order deny,allow
+        Deny from all
+        Allow from 127.0.0.1
+    </Location>
+</VirtualHost>
```

Comment:
/etc/httpd is where the directory would be on an RH system. It's going to be somewhere else (etc/apache2 I think) on a Debian-ish system. Will this be a problem?